frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/validation/PhoneNumberCodeProcessor.java

```diff
@@ -54,7 +54,7 @@ public Optional<ErrorResponse> validateCode() {
             throw new RuntimeException("Sign In Phone number codes are not supported");
         }
         var notificationType =
-                codeRequest.getJourneyType().equals(JourneyType.SIGN_IN)
+                codeRequest.getJourneyType().equals(JourneyType.PASSWORD_RESET_MFA)
```

Comment:
Is this missing the SIGN_IN case?

Reply:
sign in case is already handled in a statement above as it's not allowed. not sure why this conditional was still left in